optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/util/PlannerTestUtils.java

```diff
@@ -85,6 +89,21 @@ public Score calculateScore(Solution solution) {
 
     }
 
+    public static String readResourceToString(Class clazz, String resource) throws IOException {
```

Comment:
Use IOUtils instead, this method is obsolete.
We have 3 libraries supplying such common stuff for us: commons-lang3, commons-io and guava. One option is to use common-io's IOUtil.readLines(getClass().getResourceAsStream("..."), "UTF-8") but that returns a List<String>. IIRC, of the other util classes has a methods to return just a String.